designs/containers/vnf-deployment/vnf-deployment.md

```diff
@@ -1,44 +1,41 @@
+This document describes the design changes needed for OSC to support deployments of container VNFs on a Kubernetes(K8s) cluster.  The deployed VNF instances will be used to protect the network traffic between workload containers present on the cluster.
 
 ## Assignees
 Emanoel Xavier - https://github.com/emanoelxavier
 
 ## Background
 OSC currently supports protection of workloads hosted on VM based virtualized environments. This work expands OSC to also control security on container based environments orchestrated by [Kubernetes](#kubernetes-home). The adoption of container technologies is becoming more and more widespread due to its benefits such as easy maintainability, reusability, and minimal overhead with K8s being one of the most popular container orchestrators.  
-For the first release of the OSC containers support the focus will be mostly on the integration points between OSC, the virtualization environment (K8s) and the software defined network(SDN) controller services needed to discovering the workloads to be protected, deploying the security VNFs and performing traffic redirection through a demonstrable E2E flow.  
+For the first release of the OSC containers support the focus will be mostly on the integration points between OSC, the virtualization environment (K8s) and the software defined network(SDN) controller services needed to discovering the workloads to be protected, deploying the security VNFs and performing traffic redirection through a demonstrable E2E flow.
 
 ## Constraints and Assumptions
 
-### OVN with SFC for Kubernetes
-To perform the redirection through a container VNF using OVN the container must be on the same Kubernetes node as the protected pod. For this reason OSC may need to deploy one pod (container) per node in the cluster and ensure that new nodes will have a container VNF. To address this we should to consider using [Kubernetes Daemon Sets](#kubernetes-daemon-sets) instead of Deployments for virtualization connectors using OVN SDN. A daemon set might map to an OSC deployment spec just like a Kubernetes Deployment does but additional investigation is needed.  
-
 
 ### Nuage SDN for Kubernetes
-Nuage does not present a restriction with respect to where the container VNF should be deployed. The VNF can be deployed on any node or namespace independently of where the protected containers might be. The virtual subnet corresponding to the deployment namespace may need have some policies manually configured on Nuage though to allow for traffic to flow from another subnets/namespaces.   
+Nuage does not present a restriction with respect to where the container VNF should be deployed. The VNF can be deployed on any node or namespace independently of where the protected containers might be. The virtual subnet corresponding to the deployment namespace may need have some policies manually configured on Nuage though to allow for traffic to flow from another subnets/namespaces.

```

Comment:
>Restriction: The Nuage SDN currently requires the pod VNFs to be present on the same namespace as the protected workloads.

seems contradictory to the statement in the high-level-design. We need to add the above restriction in here